components/team_groups_manage_modal/index.js

```diff
@@ -0,0 +1,27 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
```

Comment:
This is just a helper component to implement the `ListModal` component specifically for adding groups to a team.